src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRob = {};
+
+  for (const key in robot) {
+    newRob[robot[key]] = key;
+  }
+
+  if (Object.keys(newRob).length !== Object.values(robot).length) {
+    return null;
+  }
```

Comment:
would be better to make this check in the loop to return null immediately when we find the repeated value and avoid unnecessary actions